lib/zold/node/farm.rb

```diff
@@ -165,11 +165,19 @@ def save(threads, list = [])
     def load
       @mutex.synchronize do
         if File.exist?(@cache)
-          AtomicFile.new(@cache).read.split(/\n/).map { |t| Score.parse(t) }
+          AtomicFile.new(@cache).read.split(/\n/).map { |t| parse_score_line(t)[0] }
```

Comment:
@emilianodellacasa the regex is redundant, a string would do.

Also, I don't want to care what `[0]` is.